tests/test0154.d

```diff
@@ -3,13 +3,18 @@
 //T retval:25
 // template value parameter
 
+int vpT(int I)(int c) {
+        return I - c;
+}
+
 auto foo(T U, T)() {
 	return cast(int) (U + T.sizeof);
 }
 
 int main() {
-	return foo!true() + foo!10() + foo!I();
+	return foo!true() + foo!10() + 
+		foo!I() + vpT!5(3);
```

Comment:
pass a literal and a symbol to check both work. The rest is needless complications.